app/controllers/floating_ip_controller.rb

```diff
@@ -128,51 +128,15 @@ def edit
     drop_breadcrumb(
       :name => _("Associate Floating IP \"%{address}\"") % { :address => @floating_ip.address },
       :url  => "/floating_ip/edit/#{@floating_ip.id}")
-  end
-
-  def floating_ip_form_fields
-    assert_privileges("floating_ip_edit")
-    floating_ip = find_record_with_rbac(FloatingIp, params[:id])
-    network_port_ems_ref = if floating_ip.network_port
-                             floating_ip.network_port.ems_ref
-                           else
-                             ""
-                           end
-    # TODO: router field is missing!
-    # :router_id            => floating_ip.router.id,
-    render :json => {
-      :fixed_ip_address     => floating_ip.fixed_ip_address,
-      :floating_ip_address  => floating_ip.address,
-      :cloud_network_name   => floating_ip.cloud_network.try(:name),
-      :network_port_ems_ref => network_port_ems_ref,
-      :cloud_tenant_name    => floating_ip.cloud_tenant.try(:name),
-    }
-  end
-
-  def networks_by_ems
-    assert_privileges("floating_ip_new")
-    networks = []
-    available_networks = CloudNetwork.where(:ems_id => params[:id], :external_facing => true).find_each
-    available_networks.each do |network|
-      networks << { 'name' => network.name, 'id' => network.id }
-    end
-    render :json => {
-      :available_networks => networks
-    }
+    render :new_edit
   end
 
   def new
     assert_privileges("floating_ip_new")
     @floating_ip = FloatingIp.new
     @in_a_form = true
-    @ems_choices = {}
-    network_managers.each do |network_manager|
-      @ems_choices[network_manager.name] = network_manager.id
-    end
-    drop_breadcrumb(
-      :name => _("Add New Floating IP"),
-      :url  => "/floating_ip/new"
-    )
+    drop_breadcrumb(:name => _("Add New Floating IP"), :url => "/floating_ip/new")
+    render :new_edit
```

Comment:
(and here)

Reply:
Done too.

Reply:
This is still not fixed, and neither is the other occurence on [line 131](https://github.com/ManageIQ/manageiq-ui-classic/pull/2476/files#diff-cb580b24d878da0531aed3049acfe8d4R131).

Use explicit `render :action => ...` or `render :template => ...` or create the 2 templates.